test/functional/cmdLineTests/jfr/playlist.xml

```diff
@@ -32,8 +32,11 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-ex
 			-DEXE=$(SQ)$(JAVA_COMMAND) $(JVM_OPTIONS)$(SQ) \
 			-DMETADATA_BLOB_PATH=$(Q)$(TEST_RESROOT)$(D)metadata.blob$(Q) \
 			-DRESJAR=$(Q)$(TEST_RESROOT)$(D)jfr.jar$(Q) \
-			-jar $(CMDLINETESTER_JAR) -config $(Q)$(TEST_RESROOT)$(D)jfr.xml$(Q) \
-			-explainExcludes -xids all,$(PLATFORM),$(VARIATION), -nonZeroExitWhenError; \
+			-jar $(CMDLINETESTER_JAR) \
+			-config $(Q)$(TEST_RESROOT)$(D)jfr.xml$(Q) \
+			-explainExcludes \
+			-nonZeroExitWhenError \
```

Comment:
Please include the `-plats` option here as well (to avoid surprises if any of the tests listed in `jfr.xml` need to be platform-specific).